package.json

```diff
@@ -158,6 +158,12 @@
     "uuid": "^9.0.1"
   },
   "size-limit": [
+    {
+      "brotli": false,
+      "path": "./packages/teams-js/dist/esm/packages/teams-js/src/index.js",
+      "import": "{ app, authentication, pages }",
+      "limit": "58.033 KB"
```

Comment:
>"limit": "58.033 KB" [](http://example.com/codeflow?start=6&length=20)

Is it important that this be so tightly constrained to the current size? E.g., do we want to allow for a little bit of slack in the limit (to allow for some refactoring/etc.)?

Reply:
That is something we could do. I ran into a bit of a problem trying to figure out _how much_ wiggle room we should allow, that would both allow for refactoring and small changes but still catch treeshaking breakages.